docs/how-to/authentication-plugin-examples.mdx

```diff
@@ -289,7 +313,7 @@ mutation {
 }
 ```
 
-#### Header with JWT Included (example)
+#### Header with JWT Included
 
 <!-- spell-checker: disable -->
 
```

Comment:
The section is called "Real Examples" - let's omit the Real, as all the examples should be real...